tests/nat/nat_helpers.py

```diff
@@ -0,0 +1,848 @@
+        if splited_entries:
+            num_entries = len(splited_entries[::5])
+            keys = [el.strip() for el in output_cli["stdout"].split("\n")[13].split("  ") if len(el) > 0]
+            for num in range(0, num_entries):
+                entry_values = splited_entries[(num * 5):(num * 5) + 5]
+                key = entry_values[1] if entry_values[1] != "---" else entry_values[2]
+                output[key] = {keys[i]: entry_values[i] for i in range(0, len(keys))}
+        return output
+    elif clear:
+        output_cli = exec_command(duthost, ["sudo sonic-clear nat translations"])
+        if output_cli["rc"]:
+            raise Exception('Return code is {} not 0'.format(output_cli["rc"]))
+        return output_cli["stdout"].lstrip()
+    return None
+
+
+def crud_operations_basic(duthost, flow_direction, crud_operation):
+    """
+    static NAT CLI helper
+    :param duthost: DUT host object
+    :param flow_direction: string traffic direction
+    :param crud_operation: dict dict with action and rule parameters
+    :return : dict with rule parameters
+    """
+    nat_type = "static_nat"
+    for key in crud_operation.keys():
+        output = configure_nat_over_cli(duthost, flow_direction,
+                                        crud_operation[key]["action"], nat_type,
+                                        crud_operation[key]["global_ip"], crud_operation[key]["local_ip"])
+    return output
+
+
+def crud_operations_napt(duthost, flow_direction, crud_operation):
+    """
+    static NAPT CLI helper
+    :param duthost: DUT host object
+    :param flow_direction: string traffic direction
+    :param crud_operation: dict dict with action and rule parameters
+    :return : dict with rule parameters
+    """
+    nat_type = 'static_napt'
+    for key in crud_operation.keys():
+        output = configure_nat_over_cli(duthost, flow_direction,
+                                        crud_operation[key]["action"], nat_type,
+                                        crud_operation[key]["global_ip"], crud_operation[key]["local_ip"],
+                                        proto=crud_operation[key]["proto"],
+                                        global_port=crud_operation[key]["global_port"],
+                                        local_port=crud_operation[key]["local_port"])
+    return output
+
+
+def exec_command(host, command_list):
+    """
+    method for shell execution of commands on host
+    :param host: host object
+    :param command_list: list of commands to execute
+    :return : response from host or exception __str__
+    """
+    if len(command_list) == 1:
+        try:
+            response = host.shell(command_list[0])
+            return response
+        except Exception as e:
+            return e.__str__()
+    else:
+        for command in command_list:
+            exec_command(host, [command])
+
+
+def nat_zones_config(duthost, setup_info, interface_type):
```

Comment:
What is the interface_type

Reply:
interface_type is set per module and used in parametrize fixture: 

@pytest.fixture(params=["loopback", "port_in_lag"], scope='module')
def setup_test_env(request, ptfhost, duthost, testbed):

Could be:
"loopback" which corresponds to Loopback interface in SONiC, 
"port_in_lag" which corresponds to PortChannel000X  interfaces in SONiC